providers/sparkpost/src/lib/sparkpost.provider.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ChannelTypeEnum,
+  IEmailOptions,
+  IEmailProvider,
+  ISendMessageSuccessResponse,
+} from '@notifire/core';
+
+import SparkPost from 'sparkpost';
+export class SparkpostEmailProvider implements IEmailProvider {
+  id = 'sparkpost';
+  channelType = ChannelTypeEnum.EMAIL as ChannelTypeEnum.EMAIL;
+
+  private sparkpostClient: SparkPost;
+  constructor(
+    private config: {
+      apiKey: string;
+    }
+  ) {
+this.sparkpostClient = SparkPost(config.apiKey);
+  }
+
+  async sendMessage(
+    options: IEmailOptions
+  ): Promise<ISendMessageSuccessResponse> {
+
+const response= this.sparkpostClient.transmissions.send({
+  options: {
+    sandbox: true
```

Comment:
I think this can be passed in the constructor of the provider, so if someone want's to send production emails he can control it.

Reply:
You are talking about the options, right?

Reply:
We can either just add `sandbox: boolean` to the constructor options, and later add another options on the need basis.

Reply:
Okay got it